src/config/game.json

```diff
@@ -1,31 +1,32 @@
 {
     "server":{
```

Comment:
Maybe we should split this file into server.json and units.json because the server-config is not often needed
What do you think @Phoscur ?

Reply:
It doesn't matter much. It is just needed when booting the client, it changes very rarely.
I would only split it off if that gives some kind of (performance) benefit, not seeing that yet.

Reply:
ok, I'll leave it in then